Formula/telnetd.rb

```diff
@@ -0,0 +1,63 @@
+class Telnetd < Formula
+  desc "TELNET server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-54.50.1.tar.gz"
+  sha256 "156ddec946c81af1cbbad5cc6e601135245f7300d134a239cda45ff5efd75930"
+
```

Comment:
Probably want a `keg_only :provided_pre_high_sierra` in this formula.

Reply:
@DomT4 Ok